Misc/NEWS.d/next/C API/2018-10-24-17-29-59.bpo-35059.hjUsEZ.rst

```diff
@@ -0,0 +1,2 @@
+Convert :c:func:`Py_INCREF` and :c:func:`Py_DECREF` macros to static inline
+functions.
```

Comment:
This news is confusing.  `Py_INCREF` and `Py_DECREF` are still macros.
Their implementations just use static inline function.  I don't think this change needs news entry.